graphql_compiler/tests/test_compiler.py

```diff
@@ -5298,9 +6087,17 @@ def test_multiple_outputs_in_same_fold(self):
               [x IN collected_Animal__out_Animal_ParentOf___1 | x.name] AS `child_names_list`,
               [x IN collected_Animal__out_Animal_ParentOf___1 | x.uuid] AS `child_uuids_list`
         """
+        # TODO: implement ID type for postgresql folds
```

Comment:
`ID` type in the GraphQL schema just means "GraphQL int or GraphQL string", but the handling on the Postgres side should be identical. Hopefully not too much work for a future PR?

Reply:
yep, it is simple https://github.com/kensho-technologies/graphql-compiler/blob/master/graphql_compiler/compiler/expressions.py#L927
dates/datetimes and decimals should be added too.